tools/files/darwin/com.logdna.logdna-agent.plist

```diff
@@ -15,8 +15,8 @@
         <key>RunAtLoad</key>
         <true/>
         <key>StandardErrorPath</key>
-        <string>/Library/Logs/logdna-agent.log</string>
+        <string>/var/log/logdna-agent.log</string>
```

Comment:
Nothing to do here for now, but for Homebrew best-practices this directory should be within the brew install path. By default that would be `/usr/local/var/log` but users can have a different base directory for their homebrew install. The same is true for the `ProgramArguments` path, that should also take into consideration the hombrew prefix path.

The `nginx` formula takes the approach of generating the plist at install-time for this reason 
https://github.com/denji/homebrew-nginx/blob/834eb3f2f4542d3e3ab5e9d9efd127a9cebc19cd/Formula/nginx-full.rb#L379-L401

Unrelated, line 10 above `com.logdna.logdna-agentd` - is that correct, with the extra `d` at the end?

Reply:
@mdeltito, decided to return it back - I think, after split, we will have more opportunity to follow the best practices; currently, I have decided not touch and this PR isn't place for touching this segment.

Reply:
I agree, and that sounds good. just wanted to mention it while I'm looking at it